ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -263,4 +280,7 @@ export default defineComponent({
 
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
+.pb-60px {
+  padding-bottom: 60px;
+}
```

Comment:
Maybe a bit much, try and avoid excessive bottom padding if you can. 
Ideally lean on the padding top so when you come in to add new components your not battling bottom padding from previous components. 

Could we remove this and perhaps change the `pt-4` to `py-4` on the `section` tag where this class was used. 
It might seem narrow for now but all good, we have several more components to slot in here :)

Reply:
Will do!